packages/components/addon/components/hds/pagination/nav/index.hbs

```diff
@@ -1,7 +1,7 @@
 <nav class="hds-pagination-nav" aria-label="Pagination navigation" ...attributes>
   <Hds::Pagination::Nav::ButtonArrow
     @direction="prev"
-    @type={{this.type}}
+    @hideLabel={{eq this.type "numbered"}}
```

Comment:
Thanks, this is a good update. The `type` I originally used here was actually tripping me up too since HTML buttons have their own `type` attribute.